### PR TITLE
Added the docs for the UUID polyfill

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

File: components/polyfill_uuid.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
+.. index::
+    single: Polyfill
+    single: PHP
+    single: Components; Polyfill
+
+The Symfony Polyfill / UUID Component
+========================================
```
Comment (OskarStark) [inline components/polyfill_uuid.rst]: Too long underline

File: components/polyfill_uuid.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+-----
+
+Once this component is installed in your application, you can use the following
+functions, no matter if the `PHP UUID extension`_ is installed or not in your
+server.
+
+Provided Constants
+~~~~~~~~~~~~~~~~~~
+
+* ``UUID_VARIANT_NCS`` (value = 0)
```
Comment (OskarStark) [inline components/polyfill_uuid.rst]: Maybe a table would fit better here? Maybe same for the other polyfills?

File: components/polyfill_uuid.rst (hunk 1 above)
Comment (javiereguiluz) [reply components/polyfill_uuid.rst]: Fixed. Thanks.

File: components/polyfill_uuid.rst (hunk 2 above)
Comment (javiereguiluz) [reply components/polyfill_uuid.rst]: I prefer to not do that because tables are hard to maintain in RST docs.

**Comment by OskarStark** [comment]
What about "generating" the docs for the polyfills? there is to editorial content, isn't it?

cc @weaverryan

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you Javier.

**Pull request merged**